addons/point_of_sale/i18n/point_of_sale.pot

```diff
@@ -6175,8 +6175,8 @@ msgstr ""
 #: code:addons/point_of_sale/static/src/js/Screens/PaymentScreen/PaymentScreen.js:0
 #, python-format
 msgid ""
-"The amount of your payment lines must be rounded to validate the "
-"transaction."
+"The amount of your payment lines must be rounded to validate the transaction.\n"
```

Comment:
I don't really know how the translation change should be handled in stable version, so quick question: don't you need to update the po files ? Only the pot ?

Reply:
Only the POT needs to be updated on non l10n modules